neocore/Cryptography/Crypto.py

```diff
@@ -96,7 +96,7 @@ def ToAddress(script_hash):
         Returns:
             str: base58 encoded string representing the wallet address.
         """
-        return hash_to_wallet_address(script_hash.Data)
```

Comment:
this is as the result of: https://github.com/CityOfZion/neo-python-core/pull/19/commits/8b5999b3cba7a5f81347f779564b6f6a97149105

Where we removed the redundant `hash_to_wallet_address`  function.